modules/custom/atlas/atlas_statistics/atlas_statistics.inc

```diff
@@ -166,3 +182,39 @@ function atlas_statistics_webform_is_active($nid, $date = '13 months ago') {
     return $last_submission->submitted > strtotime($date);
   }
 }
+
+/**
+ * Replicate functionality of https://github.com/drush-ops/drush/blob/8.x/commands/core/core.drush.inc#L797
+ */
+function _atlas_drush_core_requirements() {
+  include_once DRUSH_DRUPAL_CORE . '/includes/install.inc';
```

Comment:
I'm not sure where that constant is on the server. I don't see it in the 7.x repo or Express.